### ISSUE TITLE
Error running command ResizeImagesCommand for Pages with 200-500+ Images

### ISSUE THREAD

**Issue by FunkyJustin** [stevencohn/OneMore#1115]
# Problem to Solve
Error running command ResizeImagesCommand for Pages with a lot of Images

# To Reproduce

1. Go to a page with 200 - 500+ Images
2. Press CTRL + A (to select all images)
3. Right click on an empty space to show context menu
4. Click Resize and Adjust

Steps to reproduce the behavior (Alternative):
1. Go to a page with 200 - 500+ Images
2. Right click on one of the images to show context menu
3. Click Resize and Adjust

Steps to reproduce the behavior (Alternative 2):
1. Go to a page with 200 - 500+ Images
2. Go to the ribbon and Go to More (One More icon)
3. Go to images
5. Go to resize and adjust (to resize all images)
6. Click Resize and Adjust

# Expected Behavior
The Resize dialog should have appeared and I should be able to resize the image

# Environment (if applicable)

Microsoft Windows 10 Pro
Version	10.0.19045 Build 19045
Microsoft® OneNote® for Microsoft 365 MSO (Version 2308 Build 16.0.16731.20052) 32-bit 

# Screenshot of error
![image](https://github.com/stevencohn/OneMore/assets/18019897/1cd813f3-fe13-484a-841a-106141df8e52)

# Additional Context

Log:

```
04| Running command ResizeImagesCommand
04| Error running command ResizeImagesCommand
04| System.NullReferenceException
Message: Object reference not set to an instance of an object.
Source: River.OneMoreAddIn
StackTrace:    at River.OneMoreAddIn.Commands.ResizeImagesCommand.<Execute>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at River.OneMoreAddIn.CommandFactory.<Run>d__5.MoveNext()
TargetSite: [River.OneMoreAddIn] River.OneMoreAddIn.Commands.ResizeImagesCommand+<Execute>d__5::MoveNext()

```
Log 2:
```
01| Starting dllhost 4052, 3400 Mhz, 256 MB, en-US/en-US, v5.9.0, OneNote 15.0, Office 16.0, 2023-09-06 13:29
01| ..C:\Program Files (x86)\Microsoft Office\root\Office16\ONENOTE.EXE (16.0.16731.20170)
01| ..OnConnection(ConnectionMode:ext_cm_Startup,custom[1:Int32])
01| ..OneAddInsUpdate(custom[])
01| ..building ribbon
01| ..building ribbon colorizer commands
01| ..building ribbon language proofing commands
01| ..building context menu
01| ..OnStartupComplete(custom[])
01| ..starting command service
01| ..starting reminder service
01| ..starting navigation service
01| ..ready
09| Running command UpdateCommand
03| ..defined 46 hotkeys for input locale 1033
06| ..HKCR::River.OneMoreAddin SilientReminders is set to true
09| Running command ResizeImagesCommand
09| Error running command ResizeImagesCommand
09| System.NullReferenceException
Message: Object reference not set to an instance of an object.
Source: River.OneMoreAddIn
StackTrace:    at River.OneMoreAddIn.Commands.ResizeImagesCommand.<Execute>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at River.OneMoreAddIn.CommandFactory.<Run>d__5.MoveNext()
TargetSite: [River.OneMoreAddIn] River.OneMoreAddIn.Commands.ResizeImagesCommand+<Execute>d__5::MoveNext()
```

**Comment by FunkyJustin** [comment]
Note: as a means of getting around this problem for now, I am currently using the free verison of OneTastic and it has no problems resizing any number of images

**Comment by FunkyJustin** [comment]
Update: Even after going to Apps and Features to repair Microsoft OneNote and starting it up again, it is still showing the same error.

**Comment by stevencohn** [comment]
How many is "a lot", so I can try to reproduce this? Or export as a .one file and attach your page here if it's not sensitive info.

**Comment by FunkyJustin** [comment]
200 - 500+ Images

**Comment by stevencohn** [comment]
Not sure I can reproduce. When you run the Resize command, the log file should immediately start logging lines that look something like this

```
..resized from 360 x 189 to 500 x 262
..resized from 360 x 202 to 500 x 280
```

But your log file raises a NullReferenceException exception immediately. So there's something else structural on your page that I'm unaware of.

**Comment by FunkyJustin** [comment]
I'll give you this [sample page](https://mega.nz/file/15Zl1I7B#FQiDofQR6ljNjqlULwO9Dn9VG1ScOT7mTB577xBvaso) here, it's the the way I structure my page and it also triggers the same error

**Comment by stevencohn** [comment]
I see no problem with that page. I can resize one, many, or all without a problem.
I'll trudge through the code and add as much exception handling as I can but you'll not see until the next release
Sorry for the inconvenience.

**Comment by FunkyJustin** [comment]
Sorry for the confusion, I only took the few line of codes at the end in my initial post because I thought that's where it started logging the error. Here is the complete log file. 

[OneMore ResizeImageError.log](https://github.com/stevencohn/OneMore/files/12550255/OneMore.ResizeImageError.log)

And to update you, I tested the page again and the Resize Image worked and the Dialog showed up for some reason, I then copied and pasted the image and link several more times and that's where the error started again. 

Maybe it has something to do when the image has just been newly copied and pasted? (although that would contradict as I still experience the same error for my old pages with pictures and links taken from a long time ago)
